src/styles/main.scss

```diff
@@ -1,3 +1,18 @@
-body {
-  margin: 0;
+@import "./utils/_valiables.scss";
+@import "./utils/_normalize.scss";
+@import "./utils/functions";
+
+@import "./blocks/header.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/card.scss";
+
+.catalog {
+  margin: 0 auto;
+  padding: 50px 40px 72px;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  flex-wrap: wrap;
+  min-width: getMinWidth();
+  max-width: getMinWidth();
 }
```

Comment:
maybe you should create a separate file for the .catalog instead of putting it in the main